### DISCUSSION TITLE
Is ludwig video course available?

### DISCUSSION THREAD

**Discussion by PeterPirog** [ludwig-ai/ludwig#1515] [General]
I would like to ask if there is some ludwig course available online free or paid?
I know that there is documentation but sometimes I prefer video step by step tutorials.

**Comment by w4nderlust** [comment]
@PeterPirog unfortunately we don't have a video course right now, but that's a great idea!
There are some videos and blogposts that could serve as good introductions though.
Here are a few:
- Basic introduction video: https://youtu.be/OGweVw57kus 
- Full 1h presentation: https://youtu.be/xuMx-Dmdico  (it's a bit outdated, will soon record a new one after we release v0.5)
- Sentiment analysis tutorial (training from scratch): https://medium.com/ludwig-ai/the-complete-guide-to-sentiment-analysis-with-ludwig-part-i-65a9e6bc054e
- Sentiment analysis tutorial (fine-tuning): https://medium.com/ludwig-ai/the-complete-guide-to-sentiment-analysis-with-ludwig-part-ii-d9f3952a06c6
- Sentiment analysis tutorial (hyper opt): https://medium.com/ludwig-ai/hyperparameter-optimization-with-ludwig-6e31272e43fb

Hopefully these could be good starting points. Let me know if you have feedback!

(also, transforming this into a discussion rather than an issue)

**Comment by PeterPirog** [comment]
@w4nderlust , Thank you for converting issuw to the discussion.
I suggest to focus on not obvious aspects of framework like:

-  creating losses and metrics
- how to define layers parameters , mactivation function, values initialization etc.
- how to define optimizer and its parameters
- preprocessing for categorical values
-  visualization
- hyperparameters optimization
- making cross-validation
- using tensorboard to monitor hyperparameter optimization,
-  serving,
-  using ray clusters
-  CI  continous integration

because step from research environment to the development environment is huge. 

I asked about video tutorial because many times I spent a lot of hours to run some code but it become obvious when I coded with wideo tutorial step by step   ( i remember when i spend many days to learn how to build own pip package but  [one 30 minutes video explains everything).](https://www.youtube.com/watch?v=-WDV0-OB4fE&list=PLk4mwFjvagV3Y0K1RCzdzKHx3zTka9fmG&index=2)  

**I would like to stress that I thing that ludwig is the great and huge work :)**
